docroot/modules/custom/sitenow_p2lb/sitenow_p2lb.module

```diff
@@ -532,7 +532,8 @@ function sitenow_p2lb_process_paragraph($paragraph, $node) {
       $accordion_items = $paragraph->get('field_accordion_item');
       foreach ($accordion_items as $accordion_item) {
         $accordion_paragraph = $paragraph->load($accordion_item->target_id);
-        $body = $accordion_paragraph->get('field_accordion_item_body')->getValue()[0];
+        $accordionItemBody = $accordion_paragraph?->get('field_accordion_item_body')?->getValue();
+        $body = !empty($accordionItemBody) ? $accordionItemBody[0] : NULL;
```

Comment:
Let's keep our coding style consistent.
```suggestion
        $accordion_item_body = $accordion_paragraph?->get('field_accordion_item_body')?->getValue();
        $body = !empty($accordion_item_body) ? $accordion_item_body[0] : NULL;
```
Additionally, I think that you could potentially just have `$body = $accordion_paragraph->field_accordion_item_body?->value;` and it would be equivalent to this. Its possibly that `?->value` tries to get the wrong key that this is stored at, but observing it via xdebug would tell you what the key should be if its not `value`.